kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -243,6 +248,14 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
         markDuplicated(batch)
       case None =>
         val batchId = userProvidedBatchId.getOrElse(UUID.randomUUID().toString)
+        if (isResourceFromUpload) {
+          handleUploadingFiles(
+            batchId,
+            request,
+            resourceFileInputStream.get,
+            resourceFileMetadata.get.getFileName,
+            formDataMultiPartOpt)
```

Comment:
set BATCH_IMPL_VERSION to 1 if isResourceFromUpload?

Reply:
How about raising a separate PR to check or override the batch implement version ?

Reply:
cc @pan3793

Reply:
> set BATCH_IMPL_VERSION to 1 if isResourceFromUpload?

yes, we should do that for now, I'm fine to do it in place or a separate PR